doc/UserGuide.md

```diff
@@ -66,6 +66,19 @@ Examples:
 * `find Betsy Tim John`<br>
   Returns Any person having names `Betsy`, `Tim`, or `John`
 
+### Finding all persons containing any keyword in their tags: `findtag`
```

Comment:
Clear example in the User Guide, consistent with the other existing entries. Good!